### PR TITLE
Remaining LCP indicators

### PR DESCRIPTION
**Pull request by TranslucentSabre**
# Description

This change should display the origin ( if not originating from the Lancer CORE material, including the CORE Npc lcp ) for the following items:
Player Core Bonuses
Pilot Gear
Pilot Armor
Pilot Weapons
Npc Classes
Npc Features ( Systems, Weapons, Tech, Reactions )
Npc Templates

This should conclude the series of changes that I started a while ago.

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by TranslucentSabre** [comment]
The NPC display changes were not given as much care as the others. This is intentional because I know all of the NPC items are going to be looked at again for the GM re-design currently in progress.

**Comment by msprijatelj** [comment]
Pilot armor screenshot below.  I feel like the LCP text should be below the main title, *before* the divider? What do you think?
image

I understand NPC stuff is subject to change due to the GM Toolkit updates, so take the rest of my feedback with a grain of salt:

I feel like the LCP title should be a bit more faded, and with a little more padding between the NPC Class Name and the LCP title? Or maybe even right-justify the LCP title like some of the other LCP content labels?
image

It looks like the Tech/Full Tech NPC Features aren't populating the LCP source; the Traits, Systems, and Weapons are all displaying the LCP source properly (saw this with the Spite class's Tech features):
image

Comparing the NPC template format to the Core Bonus format, I'd say the LCP source name on the NPC template should have a similar "faded" look for consistency.
NPC Template:
image

Core Bonus:
image

**Comment by TranslucentSabre** [comment]
I'll try to respond to your comments as they are presented.
1) The placement of the LCP name under the divider is an attempt to put it in a common place. Only the section under the bar will be shown each time someone clicks on the gear. You can see that if you look at the different display options.
image
image
( That "UNDEFINED" prefix looks like a bug for another time )
image

I can take another look and try to figure out how the  compendium differentiates between the display modes.
I would also need to check that another way would also work when equipping in the loadout.

File: src/features/encounters/npc/new/ClassCard.vue [hunk 1]
```diff
@@ -6,6 +6,9 @@
           {{ npcc.Name }}
         </span>
       </v-col>
+      <v-col v-if="npcc.InLcp" class="ml-auto mt-n4"
+        <div class="heading h3 text--text">{{ npcc.LcpName }}</div>
```
Comment (TranslucentSabre) [inline src/features/encounters/npc/new/ClassCard.vue:10]: I played with the classes on the v-col here before in an attempt to shift it over but I can try again.

**Comment by TranslucentSabre** [comment]
Npc Tech output has been fixed. Thank you.

**Comment by TranslucentSabre** [comment]
When I get the chance I can look at text styling on these.

**Comment by TranslucentSabre** [comment]
As for putting the LCP in the title bar. That is only for the talents, and I only put it there because of the fluidity with which you can view talents in Comp/Con. They are already a problem with the LCP in the titlebar on mobile.
I don't think it would be wise to put more LCP names in titlebars due to that either.

**Comment by TranslucentSabre** [comment]
A recent comment on Discord also brought LCP downtime actions to the forefront as well. That is something that should also probably be looked at.

**Pull request merged**